src/net_processing.cpp

```diff
@@ -2891,7 +2891,7 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
 
         // Store the new addresses
         std::vector<CAddress> vAddrOk;
-        int64_t nNow = GetAdjustedTime();
+        int64_t nNow = GetTime();
```

Comment:
One side effect I’m thinking of is `if (addr.nTime <= 100000000 || addr.nTime > nNow + 10 * 60)`. If this is triggered (because our node lives in the past),  we would mark some of the recent address (including all one-hop self-announcements) to be 5 days old, and deprioritize them?